components/merkledb/src/indexes/group.rs

```diff
@@ -193,6 +193,7 @@ mod tests {
         // group.get(&3).push("quux".to_owned());
     }
 
+    #[allow(clippy::needless_pass_by_value)]
```

Comment:
Nit: If we tolerate `prepare_key_iter(&&fork)`, this warning could be solved as intended (i.e., by using `fork: &A`).